internal/txlib/pull.go

```diff
@@ -545,14 +555,43 @@ func fetchResource(
 
 func shouldSkipDownload(
 	path string, remoteStat *jsonapi.Resource, useGitTimestamps bool,
+	mode string, minimum_perc int,
 ) (bool, error) {
 	var localTime time.Time
 
+	var remoteStatAttributes txapi.ResourceLanguageStatsAttributes
+	err := remoteStat.MapAttributes(&remoteStatAttributes)
+	if err != nil {
+		return false, err
+	}
+	remoteTime, err := time.Parse(time.RFC3339,
+		remoteStatAttributes.LastUpdate)
+	if err != nil {
+		return false, err
+	}
+
+	if minimum_perc > 0 {
+		actedOnStrings := remoteStatAttributes.TranslatedStrings
+		switch mode {
+		case "reviewed", "onlyreviewed":
+			actedOnStrings = remoteStatAttributes.ReviewedStrings
+		case "proofread", "onlyproofread":
+			actedOnStrings = remoteStatAttributes.ProofreadStrings
+		}
+
+		totalStrings := remoteStatAttributes.TotalStrings
+
+		actedOnStringsPerc := (actedOnStrings * 100) / totalStrings
```

Comment:
I am a bit worried about issues caused by implicit conversions and comparisons between ints and floats. Have you encountered anything weird during testing?

For example, if the user supplies `100` meaning _"I only want fully translated RLs"_, will anything _ever_ be downloaded?

Reply:
Honestly, I wasn't so confident about this either. I'm going to make some more tests just to be sure.